src/api/environment.cc

```diff
@@ -314,6 +314,12 @@ MultiIsolatePlatform* GetMainThreadMultiIsolatePlatform() {
   return per_process::v8_platform.Platform();
 }
 
+void SetMainThreadMultiIsolatePlatform(
+    MultiIsolatePlatform* platform) {
+  per_process::v8_platform.SetPlatform(
+      static_cast<node::NodePlatform*>(platform));
```

Comment:
I’m requesting changes to at least make sure it doesn’t land with an invalid cast